meson.build

```diff
@@ -564,12 +564,13 @@ if r.returncode() == 1 and get_option('subprojects_check')
   error('Subprojects are not updated. Please run `git clean -dxff subprojects/` to delete all local subprojects directories. If you want to compile against current subprojects then set option `subprojects_check=false`.')
 endif
 
-libzip_dep = dependency('libzip', required: get_option('use_sys_libzip'), static: is_static_build)
+libzip_dep = dependency('libzip', required: get_option('use_sys_libzip'), static: is_static_build, default_options: [ 'c_std=gnu99' ])
```

Comment:
```suggestion
libzip_dep = dependency('libzip', required: get_option('use_sys_libzip'), static: is_static_build, default_options: [ 'c_std=gnu99,c99' ])
```
Add also `c99` as fallback.

Reply:
needed to make it conditional, as muon would throw an error:
```
/home/runner/work/rizin/rizin/meson.build:186:21: error ''gnu99,c99'' is not one of ['none', 'c89', 'c99', 'c11', 'c17', 'c18', 'c2x', 'gnu89', 'gnu99', 'gnu11', 'gnu17', 'gnu18', 'gnu2x']
186 |     capstone_proj = subproject('capstone-' + capstone_version, default_options: ['default_library=static', 'c_std=gnu99,c99'])
                          ^_________________________________________________________________________________________________________
```